src/main/java/entities/Recipe2.java

```diff
@@ -0,0 +1,32 @@
+package entities;
+import entities.Ingredient;
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+public class Recipe {
+    /**
+     * recipe object to be instantiated by recipeBuilder class.
+     * Information regarding a recipe is then collected in a dictionary within the instance
+     */
+     private Dictionary<String, Object> dict = new Hashtable();
```

Comment:
This shouldn't be declared like this. The instance variable should be created in the constructor of the Recipe Class.